examples/events.js

```diff
@@ -11,18 +9,15 @@ async function main () {
   let db
 
   try {
-    const ipfs = await IPFS.create({
+    const ipfs = await create({
       repo: './orbitdb/examples/ipfs',
       start: true,
       EXPERIMENTAL: {
         pubsub: true,
       },
     })
-    const orbitdb = await OrbitDB.createInstance(ipfs, {
-      directory: './orbitdb/examples/eventlog'
-    })
-    db = await orbitdb.eventlog('example', { overwrite: true })
-    await db.load()
+    const orbitdb = await OrbitDB({ ipfs, directory: './orbitdb/examples/events' })
```

Comment:
We could prolly remove the `/events` part from the path just to make it a little shorter (and not to confuse with the directory option being smth in this repo)?